tests/test_ble_interface_core.py

```
@@ -388,29 +453,1263 @@ def test_state_manager_closing_only_for_disconnect() -> None:
+    iface = _build_interface(monkeypatch, DummyClient(), start_receive_thread=False)
+
+    with pytest.raises(BLEInterface.BLEError, match=ERROR_MANAGEMENT_ADDRESS_EMPTY):
+        getattr(iface, method_name)("   ")
+
+    iface.close()
+
+
+@pytest.mark.parametrize("method_name", ["pair", "unpair"])
+@pytest.mark.parametrize(
+    "invalid_timeout",
+    [None, 0.0, -1.0, float("nan"), float("inf"), cast(Any, True)],
```

Ruff (0.15.4)

[error] Boolean positional value in function call

(FBT003)